diff -Nru cdk-2.1.1-old/base/standard/src/main/java/org/openscience/cdk/graph/invariant/Canon.java cdk-2.1.1-new/base/standard/src/main/java/org/openscience/cdk/graph/invariant/Canon.java
--- cdk-2.1.1-old/base/standard/src/main/java/org/openscience/cdk/graph/invariant/Canon.java	2017-12-28 21:00:47.000000000 +0100
+++ cdk-2.1.1-new/base/standard/src/main/java/org/openscience/cdk/graph/invariant/Canon.java	2018-01-20 21:36:05.752123215 +0100
@@ -310,6 +310,7 @@
             int expH = 0;
             int elem = atomicNumber(atom);
             int chg = charge(atom);
+            int mass = atom.getMassNumber() != null ? atom.getMassNumber() : 0;
 
             // count non-suppressed (explicit) hydrogens
             for (int w : graph[v])
@@ -327,6 +328,8 @@
             label |= Math.abs(chg) & 0x3;
             label <<= 4; // hydrogen count <= 15 (4 bits)
             label |= impH + expH & 0xf;
+            label <<= 10;
+            label |= mass;
 
             labels[v] = label;
         }
diff -Nru cdk-2.1.1-old/base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java cdk-2.1.1-new/base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java
--- cdk-2.1.1-old/base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java	2017-12-28 21:00:47.000000000 +0100
+++ cdk-2.1.1-new/base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java	2018-01-22 00:01:35.641260386 +0100
@@ -223,7 +223,7 @@
                         IBond bond = bondMap.get(v, w);
                         if (w > v && bond.getOrder() == IBond.Order.DOUBLE) {
                             if (centers.elementType(w) == Stereocenters.Type.Tricoordinate
-                                && centers.isStereocenter(w) && !isInSmallRing(bond)) {
+                                && centers.isStereocenter(w) /*&& !isInSmallRing(bond)*/) {
                                 IStereoElement element = createGeometric(v, w, centers);
                                 if (element != null) elements.add(element);
                             }
